src/status_im/ui/screens/syncing/views.cljs

```diff
@@ -0,0 +1,34 @@
+(ns status-im.ui.screens.syncing.views
```

Comment:
perhaps we can directly move this into status_im2. 

imo we should not be adding new files into status-im.

@OmarBasem , @ilmotta  @flexsurfer wdyt?

Reply:
you are right, I will look into moving this into status_im2, should not be a big deal to do that.. thanks

Reply:
yes please

Reply:
but not just move, you shouldn't require old namespaces, and use only new one